docs/code-reviews.md

```diff
@@ -1,4 +1,70 @@
+merge your branches frequently. A user makes a branch to change a feature or adding some new analysis and make several commits to the new branch. When the time comes to merge this back to master, there is a natural inflexion point to do a code review, and each merge should be 
+accompanied with a code review. If you are in a hurry, and no-one has time to do a more thorough 
+code review, bear in mind that even a cursory code review is better than no code review. 
+
+Merging a new branch should not be your only trigger for a review. If you are stuck or unsure what direction to take, a code review can clarify what path to take. In general, the more frequent the reviews, the better quality the code tends to be. 
+
+## How can I write a good Pull Request?
+Some best practice gathered from current users of OpenSAFELY are described below:
+
+ - Thinking about the purpose of the review can be helpful as this may 
+vary from review to review. For example, are you looking for a check of the implementation of the logic or methods 
+outlined in the protocol (in which case, linking the protocol can aid this), or are you looking for a 
+review of how well you document your code and if it is clear and understandable? Being transparent 
+will help your reviewer do a better job quicker. 
+- Good descriptive names for Pull Requests and commits help the reviewer follow the logic of the code and the changes made (see example of commits below)
+- Multiple smaller Pull Requests are easier than one large 
+- Code comments and documentations make the logic easier to follow 
+- Naming the order of the files, for example, `01_Data_Extraction.py`, `02_Data_Cleaning.py`, etc. 
+- Allow sufficient time for the review
+
+#### Examples of good commit messages
+![image](https://user-images.githubusercontent.com/25401512/109499651-166aaf80-7a8d-11eb-86de-f4a0b549fc90.png)
+
+## How can I be a good reviewer?
+Some best practices gathered from current users of OpenSAFELY are described below: 
+
+- Code review should be a collaborative conversation and don't be afraid to check assumptions or clarify intentions within the review (you can add messages to the end of the review request). 
+- Actionable feedback with suggestions is helpful. 
+- It can be helpful to highlight the levels of code review comments. For example, 
+if something is nice-to-have-but-not-important, you can prefix it with `Nit:`. For example, 
+`Nit: I think this would be cleared if you didn't use an underscore in this variable name`
+- Being clear about what exactly you have reviewed or how thorough the review was—for example, being clear if you read through the code but did not run it (and therefore may miss running errors in implementation). Unit tests should catch that the code is runnable, but will not necessarily check it is correct.  
```

Comment:
Can you put spaces around the emdash, e.g. `was — for`, otherwise it looks like it's a hyphenated word "was-for" :)